Source/Core/Core/State.cpp

```diff
@@ -722,8 +682,8 @@ static bool ValidateHeaders(const StateHeader& header)
     return false;
   }
 
-  // Check both the state version and the revision string
-  std::string current_str = Common::GetScmRevStr();
+  // Check the state version.
+  // FYI: revision string isn't currently tested, that might make things annoying.
```

Comment:
Might make what annoying? The revision string was added just for the purpose of showing it to the user if Dolphin was unable to load the savestate due to the version integer not matching.

Reply:
The comment implied save states would fail to load if the revision string in the file didn't match the one in the application. That was not and is still not the case, `current_str` was an unused variable.

Failing to load states on revision string mismatch would be majorly inconvenient.

Reply:
Oh, you mean it would be annoying if the revision string was tested. That makes sense now that I read it like that. I thought you meant that not testing the revision string would make things annoying in the code somehow.

Reply:
Gotcha, I'll make the comment less ambiguous.